src/server/worker/datum/datum.proto

```diff
@@ -5,6 +5,7 @@ option go_package = "github.com/pachyderm/pachyderm/v2/src/server/worker/datum";
 
 import "gogoproto/gogo.proto";
```

Comment:
can you import this in `src/internal/task/taskprotos/register.go` so `ListTask` can unmarshal them? Maybe it's not a great strategy